libethcore/SealEngine.cpp

```diff
@@ -66,7 +66,7 @@ void SealEngineFace::verify(Strictness _s, BlockHeader const& _bi, BlockHeader c
 {
     _bi.verify(_s, _parent, _block);
 
-    if (_s != CheckNothingNew)
+    if (_s != CheckNothingNew && _s != JustSeal)
```

Comment:
It doesn't seem to me that `JustSeal` works right (e.g. `Ethash` validates seal only if `_s == CheckEverything`)
It looks to be used only in json tests, do you actually need it? Couldn't you use `CheckNothingNew` ?

Reply:
I could. But logically those checks should not be performed for JustSeal and for lots of other enums as well

Reply:
I propose to remove `JustSeal` from enum then.